dataflow/src/main/java/org/checkerframework/dataflow/cfg/AbstractCFGVisualizer.java

```diff
@@ -0,0 +1,337 @@
+package org.checkerframework.dataflow.cfg;
+
+import java.util.*;
+import javax.lang.model.type.TypeMirror;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.*;
+import org.checkerframework.dataflow.cfg.block.*;
+import org.checkerframework.dataflow.cfg.node.Node;
+
+/**
+ * An abstract class of control flow graph visualizer. To achieve this abstract class, you need to
```

Comment:
Highlight that `generateNodes` is a new abstract method that needs to be implemented.
Also mention the new `Helper` methods that simplify writing an implementation of other methods.